pycbc/events/stat.py

```diff
@@ -579,9 +621,8 @@ def logsignalrate(self, stats, shift, to_shift):
                     self.param_bin[ref_ifo][loc] != nbinned
                 )[0]
                 rate[rtype[missed]] = self.max_penalty
-
-            # Scale by signal population SNR
-            rate[rtype] *= (sref / self.ref_snr) ** -4.0
+                # Scale by signal population SNR
+                rate[rtype] *= (sref[rtype] / self.ref_snr) ** -4.0
```

Comment:
I am not seeing what this change is doing .. & how is sref an object that can be indexed?

Reply:
@tdent This isn't really a change at all. For the two-detector case this call is now done in the cython code, so this call is moved up a level to only be done in the [unoptimized] three-detector code. The [rtype] index was always there, if you compare line 516 (old code) to 536 (new code) you see that I am now not applying the rtype index so it just moves to here.

sref is an array (it stores the SNR in the reference ifo for all triggers).

Reply:
ah, this is inside the loop over ifos.